### ISSUE TITLE
solved: Re-install fail

### ISSUE THREAD

**Issue by Werner-G** [hobbyquaker/node-red-contrib-zigbee#78]
My installation is running on a RPi. I had some issues with that node and tried to deinstall the node completely. But still some relicts are remaining and prevent a new fresh install. My question is: which folders / files do I have to delete manually to have a clean system?

**Comment by tentsubore** [comment]
I don't need this module as I found a way to achieve what I need. It started causing my nodered to crash (some error getEntries is null) so I tried to uninstall it but was unable to because it said the herdsman in use even though the config node is clear and I disabled it in the config node. 

So I tried uninstalling it using npm, now the crashes stop but it is now spitting an error every 5s. 

```
11/30/2019, 12:17:35 AMnode: zigbee herdsman
msg : string[95]
"Error while opening serialport 'Error: Error Resource temporarily unavailable Cannot lock port'"
```

I probably didn't uninstall it the right way. How do I fix this?
